packages/apify/src/input-schemas.ts

```diff
@@ -0,0 +1,89 @@
+// TODO: https://github.com/apify/apify-shared-js/issues/547
+
+import { existsSync, readFileSync } from 'node:fs';
+import { join } from 'node:path';
+import process from 'node:process';
+
+import type { Dictionary } from '@crawlee/utils';
+
+// These paths are used *if* there is no `input` field in the actor.json configuration file!
+const DEFAULT_INPUT_SCHEMA_PATHS = [
+    ['.actor', 'INPUT_SCHEMA.json'],
+    ['INPUT_SCHEMA.json'],
+    ['.actor', 'input_schema.json'],
+    ['input_schema.json'],
+];
+
+const ACTOR_SPECIFICATION_FOLDER = '.actor';
+
+const LOCAL_CONFIG_NAME = 'actor.json';
+
+const readJSONIfExists = (path: string): Dictionary | null => {
+    if (existsSync(path)) {
+        const content = readFileSync(path, 'utf8');
+        return JSON.parse(content);
+    }
+
+    return null;
+};
+
+export const kNoActorInputSchemaDefined = Symbol.for(
```

Comment:
this is really weird naming. also, do we really need to export this symbol? if so, it should be marked as `@ignore` so it wont end up in the docs

Reply:
I just used naming similar to what I've seen in node (and we need it exported to check for it in actor.ts). I'll slap on the ignore tag in a sec

Reply:
i think we already discussed this some time ago. its weird, confusing, and unnecessary, lets not adopt it just because people maintaining node use it.

btw it looks like this file is not exported from the barrel so its fine i guess